aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -46,8 +48,9 @@ Tensor _add_out(Tensor& out, const Tensor& self, const Tensor& other) {
 
 template <bool ReLUFused = false>
 Tensor _add_scalar_out(Tensor& out, const Tensor& self, Scalar other) {
-  TORCH_CHECK(self.qscheme() == kPerTensorAffine,
-              "Only per tensor affine is supported for now!!");
+  TORCH_CHECK(
+      self.qscheme() == kPerTensorAffine,
+      "Only per tensor affine is supported for now!!");
```

Comment:
@dskhudia I think the code in the comment below is no longer relevant with your changes. Can you update that as well to reflect the new requant flow?

Reply:
Thanks. Good catch. I forgot about it.